qiita_pet/support_files/doc/source/analyzingsamples/index.rst

```diff
@@ -512,3 +512,526 @@ Beta Correlation
   * **Pearson** :ref:`[56]<reference56>` : Measures how strong the linear relationship is between 2 variables
 
 * Gives scatterplot of the distance matrix on the x-axis and the variable being tested on the y-axis
+
+-------------------------------------------------------------------------------
+
+Statistical Analysis to Justify Clinical Trial Sample Size Tutorial
+-------------------------------------------------------------------
+
+The goal of this tutorial is to demonstrate how to analyse public data similar to that one may obtain from one’s own proposed study; and use this to find the minimum sample size needed for appropriate/sufficient statistical power. This will allow relevant conclusions to be drawn for the minimum clinical trial size in one’s actual (own) study. The information obtained using this public data will therefore allow justification of the clinical trial format, and strengthen e.g. grant applications.
```

Comment:
I guess this is a Windows thing but not sure how the `’` in `one’s` will work vs. a regular `'`

Reply:
I have replaced all of these with find and replace.